src/main/java/tests/OperationStrategyTest.java

```diff
@@ -0,0 +1,25 @@
+package tests;
+
+import core.basesyntax.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+
+public class OperationStrategyTest {
+    @Test
+    public void testOperationStrategy() {
+        Map<Operation, OperationHandler> operationHandlers = new HashMap<>();
```

Comment:
According to the SOLID principles and to ensure the independence of services, consider injecting the `operationHandlers` map through the constructor of the `OperationStrategyImpl` class instead of creating it inside the test method. This way, you can easily switch between different implementations without changing the test code.